src/analysis/modals/ComputeModal/AzureComputeModal/AzureComputeModal.test.ts

```diff
@@ -68,9 +78,7 @@ describe('AzureComputeModal', () => {
 
   beforeEach(() => {
     // Arrange
-    Ajax.mockImplementation(() => ({
-      ...defaultAjaxImpl,
-    }));
+    asMockedFn(Ajax).mockReturnValue(defaultAjaxImpl);
   });
 
   afterEach(() => {
```

Comment:
pretty sure we don't need jest.clearAllMocks on afterEach anymore.... we have this configured to always happen now....